src/main/java/com/denizenscript/depenizen/bukkit/events/shopkeepers/ShopKeeperTradeScriptEvent.java

```diff
@@ -14,15 +13,14 @@ public class ShopKeeperTradeScriptEvent extends BukkitScriptEvent implements Lis
 
```

Comment:
fix the classname/filename/event name too. Ideally git would recognize that "Completed" is a rename of the old event rather than thinking "initiated" is an edit of the old event (not required cause git does what git wants)